content/reference/dos-mitigation.md

```diff
@@ -252,6 +265,13 @@ or by setting the environment variable `GOLOG_LOG_LEVEL="canonical-log=info"`.
 In rust-libp2p you can do something similar yourself by logging a sample of
 connection events from [SwarmEvent](https://docs.rs/libp2p/latest/libp2p/swarm/enum.SwarmEvent.html).
 
+js-libp2p logs to stdout based on the contents of the `DEBUG` environmental variable. This can be set
+to log everything with `DEBUG=*`, individual components `DEBUG=libp2p:connection-manager*,libp2p:upgrader*`.
+
+If the logs are too verbose you can also exclude components trace logging `DEBUG=libp2p:*,-*:trace` and
```

Comment:
We should include a paragraph explaining how to do this in the browser. I found it quite tricky.

Reply:
Done!  Might also add it to the `GETTING_STARTED.md` in the js-libp2p repo docs folder.